contracts/eos/BancorConverter/BancorConverter.hpp

```diff
@@ -444,6 +419,32 @@ class [[eosio::contract]] BancorConverter : public contract { /*! \endcond */
         double asset_to_double( const asset quantity );
         asset double_to_asset( const double amount, const symbol sym );
 
+        // log
+        void EMIT_CONVERSION_EVENT(
```

Comment:
can just use normal function names here, no need for the old format anymore.
`emit_conversion_event` etc.?

Reply:
👍